include/irr/asset/CTexturePacker.h

```diff
@@ -75,18 +176,10 @@ class CTexturePacker
     }
 
 public:
-    struct STexOffset
-    {
-        uint16_t x, y;
-    };
-
-    CTexturePacker(E_FORMAT _format, uint32_t _pgSzxy = 256u, uint32_t _tilesPerDim = 32u, uint32_t _numLayers = 4u) :
-        m_pgSzxy(_pgSzxy),
-        m_tilesPerDim(_tilesPerDim),
-        m_addrAlctr(14u, _tilesPerDim)
+    ICPUTexturePacker(E_FORMAT _format, uint32_t _pgSzxy = 256u, uint32_t _tilesPerDim = 32u, uint32_t _numLayers = 4u) :
+        ITexturePacker(_pgSzxy, _tilesPerDim, _numLayers)
     {
         assert(core::isPoT(_tilesPerDim));
-        assert(core::isPoT(_pgSzxy));
```

Comment:
yeah now page size needs to be PoT due to mip-tail issues, however `_tilesPerDim` only need to be a multiple of 2^`(maxMipLevel-pageTileMipTailSize)`